### ISSUE TITLE
Dapple 0.8.33 does not execute any tests

### ISSUE THREAD

**Issue by rudolfix** [nexusdev/dapple#365]
No tests are executed on my code. I checked one of your repositories to make sure I didn't break anything:
https://github.com/nexusdev/ds-roles which contains single test
dapple test says
Testing..
Summary
  Passed all 0 tests!

I ran those tests in a container where I had no problems with previous versions.
btw. when I recreate a container with 0.8.28 I'm getting this error on dapple build. It's getting tricky!
```
TypeError: state.workspace.buildDappfileTree is not a function
    at export_sol (/usr/lib/node_modules/dapple/node_modules/dapple-core/export.js:9:31)
    at DestroyableTransform._flush (/usr/lib/node_modules/dapple/node_modules/dapple-script/lib/main.js:35:26)
    at DestroyableTransform.<anonymous> (/usr/lib/node_modules/dapple/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:115:49)
    at DestroyableTransform.g (events.js:291:16)
    at emitNone (events.js:86:13)
    at DestroyableTransform.emit (events.js:185:7)
    at prefinish (/usr/lib/node_modules/dapple/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:465:12)
    at finishMaybe (/usr/lib/node_modules/dapple/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:473:7)
    at endWritable (/usr/lib/node_modules/dapple/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:485:3)
    at DestroyableTransform.Writable.end (/usr/lib/node_modules/dapple/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:455:41)
```

**Comment by nmushegian** [comment]
encountering this in some repos as well

**Comment by rudolfix** [comment]
BTW. TypeError is caused by erroneous dependencies in package.json. dapple-core is linked like this
"dapple-core": "^0.1.26",
this will cause to link the newest version which is incompatible with the 0.1.26. so ^ is not correctly used...

**Comment by rudolfix** [comment]
fixed in dapple 0.8.35
